core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceMediaPlayer.java

```diff
@@ -322,7 +322,7 @@ protected void smartMarkAsPlayed(Playable media) {
 
         void onBufferingUpdate(int percent);
 
-        void reloadUI();
+        void reloadUI(boolean onlyMediaSession);
```

Comment:
I'm not sure if I understand the variable name 'onlyMediaSession'.  Is there a better one you could use?  Could you add a comment explaining what it's for?